src/main/java/frc/robot/commands/wrist/WristIntakeRelease.java

```diff
@@ -27,14 +25,13 @@ public WristIntakeRelease(WristIntake intake) {
         addRequirements(intake);
 
         StartEndCommand startMotors =
-            new StartEndCommand(() -> intake.setMotors(Constants.Wrist.INTAKE_RELEASE_SPEED),
-                () -> intake.setMotors(Constants.Wrist.INTAKE_STOP_SPEED));
-        ConditionalCommand condition =
-            new ConditionalCommand(startMotors, new InstantCommand(), Scoring::gamePieceIsCube);
-        SequentialCommandGroup openGrabber = new SequentialCommandGroup(new WaitCommand(.25),
-            new InstantCommand(() -> intake.openGrabber()));
+            new StartEndCommand(() -> intake.setMotor(Constants.Wrist.INTAKE_RELEASE_SPEED),
+                () -> intake.setMotor(Constants.Wrist.INTAKE_STOP_SPEED));
+        // ConditionalCommand condition =
+        // new ConditionalCommand(startMotors, startMotors, Scoring::gamePieceIsCube);
+        SequentialCommandGroup openGrabber = new SequentialCommandGroup(new WaitCommand(.25));
```

Comment:
I don't think this is necessary anymore?? This can probably be converted to just extend `CommandBase` instead of `ParallelCommandGroup`.  I don't think u guys are doing different things based on game piece either now??

Reply:
Converted to CommandBase in latest commits